include/net/nrf_cloud.h

```diff
@@ -551,12 +556,14 @@ int nrf_cloud_tenant_id_get(char *id_buf, size_t id_len);
  *
  * @param[in] time_valid_s How long (seconds) the JWT will be valid. Maximum
  *                         duration specified by @ref NRF_CLOUD_JWT_VALID_TIME_S_MAX.
- *                         If zero, NRF_CLOUD_JWT_VALID_TIME_S_MAX will be used.
+ *                         If zero, NRF_CLOUD_JWT_VALID_TIME_S_DEF will be used.
  * @param[in,out] jwt_buf Buffer to hold the JWT.
  * @param[in] jwt_buf_sz  Size of the buffer (recommended size >= 600 bytes).
  *
- * @retval 0 If successful.
- *           Otherwise, a (negative) error code is returned.
+ * @retval 0     JWT generated successfully.
+ * @retval ETIME Generated JWT has no expiration because the modem does not have valid date/time.
+ * @warning A JWT with no expiration could present a security risk, use with caution.
```

Comment:
Looks like a warning can't be associated with a `retval`, in the generated document the warning is after the description:

http://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-5974/nrf/libraries/networking/nrf_cloud.html?highlight=nrf_cloud_jwt_generate#c.nrf_cloud_jwt_generate

Maybe the "warning" should be moved after "brief" because that's how it's also in the generated documentation? And should the return value be also mentioned in the text?